README.md

```diff
@@ -448,6 +448,11 @@ lists pre-compiled ``protoc`` artifacts that can be used by this plugin.
 
 ### IntelliJ IDEA
 
+Be sure to [delegate builds to Gradle](https://www.jetbrains.com/idea/whatsnew/#v2016-3-gradle)
```

Comment:
This link doesn't take me to the expected content.

Reply:
Unfortunately there's no direct link to that section, but it is this feature:
Coverage for Gradle

IntelliJ IDEA 2017.3 lets you run tests with coverage by using the Gradle Test Runner. You can even run tests with coverage if you select the Delegate IDE build/run action to Gradle option. Choose how you want to run your test with coverage from the main editor: with the platform test runner or with the Gradle Test Runner.